enterprise/web.el/src/org/netbeans/modules/web/el/completion/ELRawObjectPropertyCompletionItem.java

```diff
@@ -51,12 +53,8 @@ public ElementKind getKind() {
     }
 
     @Override
-    public String getLhsHtml(HtmlFormatter formatter) {
-        ElementKind kind = getKind();
-        formatter.name(kind, true);
-        formatter.appendText(getName());
-        formatter.name(kind, false);
-        return formatter.getText();
+    public String getRhsHtml(HtmlFormatter formatter) {
+        return clazz;
```

Comment:
Please use `formatter.appendText` and then `formatter.getText`. At this time only raw types are supported, but in the future generics might come in and they might/will trip the HTML renderer, as the `<` will yield tag handling.